console-subscriber/src/aggregator/mod.rs

```diff
@@ -767,7 +772,11 @@ impl Aggregator {
             } => {
                 let resource_id = self.ids.id_for(resource_id);
                 if let Some(mut stats) = self.resource_stats.update(&resource_id) {
-                    let upd_key = (&update.field).into();
+                    let field_name = update.field.name.clone().expect("field misses name");
```

Comment:
hmm, i'm not sure if we really want to panic here...this should maybe just log a warning and then bail?
```suggestion
                    let field_name = match update.field.name.clone() {
                        Some(name) => name,
                        None => {
                            tracing::warn!(?update.field, "field missing name, skipping...");
                            return;
                        }
                    };
```